.github/dependabot.yml

```diff
@@ -9,3 +10,24 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  # keycloak UI dependencies
```

Comment:
I don't see the need to add a comment here, it's pretty clear from the path where this is being applied.
```suggestion
```